op/model.py

```diff
@@ -281,3 +309,9 @@ def relation_set(self, relation_id, key, value):
 
     def config_get(self):
         return self._run('config-get')
+
+    def local_unit_name(self):
+        return os.environ['JUJU_UNIT_NAME']
+
+    def local_app_name(self):
+        return self.local_unit_name().split('/')[0]
```

Comment:
In the constructor:
```
self.local_unit_name = os.environ['JUJU_UNIT_NAME']
self.local_app_name = self.local_unit_name.split('/')[0]
```
I actually find the "local" term a bit vague for this purpose, but let's keep going since this is trivial to change later once we find a better term.